src/api.ts

```diff
@@ -11,6 +11,8 @@ interface Env {
 // env object is injected by JSVM
 declare let env: Env;
 
+// TODO: rewrite api.js to named parameters?
+// TODO: should have the same interface as console.log
```

Comment:
Good catch! `log` can actually works on `message: any` instead of `string`: https://github.com/near/near-sdk-js/blob/cb152ebb16e4318207cc0a1b0507b6e6ecbeb3da/tests/src/log_panic_api.js#L9

So we can just modify the interface here

Reply:
I see! Your enhanced `near.log` is awesome!